terraform/manager.tf

```diff
@@ -235,7 +235,11 @@ write_files:
       fi
     path: /root/manager.sh
     permissions: 0700
+packages:
+  - docker-ce-cli
+  - docker-compose
```

Comment:
This is not necessary. The missing Docker resulted from a bug in the Bootstrap playbook.

https://github.com/osism/testbed/pull/546

Reply:
Hmm, I still saw a missing docker with recent testing ...

Reply:
Then it is a bug elsewhere since Docker is installed by an Ansible role. If Docker is missing this role has failed.

Reply:
ansible was failing on the OTC images due to ansible-2.9 being preinstalled there.
The dpkg -r ansible is all that's needed.